src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -19,11 +19,11 @@
 import seedu.address.commons.util.CollectionUtil;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
+import seedu.address.model.person.information.Address;
+import seedu.address.model.person.information.Email;
+import seedu.address.model.person.information.Name;
```

Comment:
I like how you moved address email and names into their own package to keep it well organised